src/components/SubmitEvent.tsx

```diff
@@ -34,24 +34,20 @@ const SubmitEvent = ({
   };
 
   const formNotValid = (): boolean => {
-    const valueMissing: boolean = Object.values(formData).some(
-      (value: string) => {
-        if (value === "") {
-          alert("Veuillez remplir tous les champs");
-          return true;
-        }
-        return false;
-      }
-    );
```

Comment:
I prefer breaking up into two steps.  That way the `.some()` filter function is much more readable and then it's easier to see from the outside when you want to raise an `alert`